site/gatsby-site/src/components/ui/PopOvers.js

```diff
@@ -89,3 +89,5 @@ export const tags = Popover(
   'Tag the type of incident report',
   `The purpose of report tags is to support sorting reports on the citation page for the incident. When in doubt, you can leave this field blank and a database editor will select appropriate tags at ingestion time.`
 );
+
+export const language = Popover('Language of the incident report', `????`);
```

Comment:
@smcgregor help here, please.

Reply:
`This is the written language of the text found in the report, not the language of the people or countries involved in the report. The reports will be translated to supported languages when we ingest the text.`